src/target/xtensa/xtensa.h

```diff
@@ -187,13 +204,21 @@ struct xtensa_sw_breakpoint {
  */
 struct xtensa {
 	unsigned int common_magic;
-	const struct xtensa_config *core_config;
+	struct xtensa_chip_common *xtensa_chip;
+	struct xtensa_config *core_config;
 	struct xtensa_debug_module dbg_mod;
 	struct reg_cache *core_cache;
-	unsigned int regs_num;
+	uint32_t total_regs_num;
```

Comment:
For the internal OpenOCD variables unsigned int preferred over the uint32_t. same as int for the int32_t

Reply:
Done for this and below uint32_t vars, but left spill_loc as that is a 32-bit target address